app/views/user_mailer/registration_successful.html.erb

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml">
```

Comment:
@josephelias-rubyians  Shall we try bootstrap-email gem for this to style better?